src/v/debug_bundle/tests/debug_bundle_service_test.cc

```diff
@@ -64,3 +82,193 @@ TEST_F_CORO(debug_bundle_service_fixture, bad_rpk_path) {
       [](debug_bundle::service& s) -> ss::future<> { co_await s.start(); }));
     EXPECT_TRUE(co_await ss::file_exists(debug_bundle_dir.native()));
 }
+
+TEST_F_CORO(debug_bundle_service_started_fixture, run_process) {
+    using namespace std::chrono_literals;
+
+    debug_bundle::job_id_t job_id(uuid_t::create());
+
+    auto res = co_await _service.local().initiate_rpk_debug_bundle_collection(
+      job_id, {});
+    ASSERT_FALSE_CORO(res.has_failure()) << res.as_failure().error().message();
+
+    auto status = co_await _service.local().rpk_debug_bundle_status();
+    ASSERT_FALSE_CORO(status.has_failure())
+      << res.as_failure().error().message();
+
+    EXPECT_EQ(
+      status.assume_value().status, debug_bundle::debug_bundle_status::running);
+    EXPECT_EQ(status.assume_value().job_id, job_id);
+    EXPECT_EQ(status.assume_value().file_name, fmt::format("{}.zip", job_id));
+
+    co_await ss::sleep(6s);
```

Comment:
any chance at all to flake here? since it's a test, maybe we could just busy wait on the status we want with a more generous failure timeout?

Reply:
yeah good point, I should do that.